test/data/register-a-death-questions-and-responses.yml

```diff
@@ -31,5 +32,6 @@
 - libya # :consular_service_fees_libya
 - papua-new-guinea # :embassy_high_commission_or_consulate "British high commission"
 - philippines # :post_only_#{philippines}
+- somalia
 - south-georgia-and-south-sandwich-islands # :overseas_passports_embassies (no FCO organisation)
 - taiwan # :embassy_high_commission_or_consulate "British Trade & Cultural Office"
```

Comment:
[My previous comment](https://github.com/alphagov/smart-answers/pull/2635/commits/78b209dba7a258ebd1e2b9f474a85df093ba1e6d#r70607811) also applies to this commit.